app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Admin::UsersController < ApplicationController
```

Comment:
this should be inherited from `Admin::ApplicationController`

Reply:
i wrote as you said.